### ISSUE TITLE
Unicode errors when running playbook with unicode under ansible 1.9.6

### ISSUE THREAD

**Issue by simfarm** [ansible/ansible#16373]
##### ISSUE TYPE
- Bug Report
##### ANSIBLE VERSION

```
[root@ip-10-65-202-196 _3120__ansible_playbooksgit_j]# ansible --version
ansible 1.9.6
  configured module search path = None
```
##### CONFIGURATION

All defaults.
##### OS / ENVIRONMENT

NA
##### SUMMARY

"UnicodeEncodeError" when running playbook with unicode.
##### STEPS TO REPRODUCE
- Create this [playbook](https://github.com/jlaska/ansible-playbooks/blob/master/utf-8-%E4%89%AA%E1%89%92%EC%B9%B8%E2%B1%B7%EA%AF%94%E5%99%82%ED%8F%84%E8%94%86%E3%AA%97%E8%BC%A5.yml), and call it `utf-8-䉪ቒ칸ⱷꯔ噂폄蔆㪗輥.yml`.
- Use the following inventory file:

```
[local]
127.0.0.1 ansible_connection=local ansible_python_interpreter='/usr/bin/env python'
```
##### EXPECTED RESULTS

Successful playbook run.
##### ACTUAL RESULTS

```
[root@ip-10-65-202-196 _3120__ansible_playbooksgit_j]# ansible-playbook -i inventory utf-8.yml 
Traceback (most recent call last):
  File "/usr/bin/ansible-playbook", line 324, in <module>
    sys.exit(main(sys.argv[1:]))
  File "/usr/bin/ansible-playbook", line 264, in main
    pb.run()
  File "/usr/lib/python2.6/site-packages/ansible/playbook/__init__.py", line 310, in run
    play = Play(self, play_ds, play_basedir, vault_password=self.vault_password)
  File "/usr/lib/python2.6/site-packages/ansible/playbook/play.py", line 194, in __init__
    self._tasks      = self._load_tasks(self._ds.get('tasks', []), load_vars)
  File "/usr/lib/python2.6/site-packages/ansible/playbook/play.py", line 606, in _load_tasks
    tokens = split_args(str(x['include']))
UnicodeEncodeError: 'ascii' codec can't encode characters in position 65-74: ordinal not in range(128)
```

**Comment by abadger** [comment]
Made a PR which should fix this on 1.9.x.  Also tested devel and the problem is not present there.

**Comment by abadger** [comment]
Merged to stable-1.9.  There are no plans for a 1.9.7 release but if one becomes necessary this fix will be in it.
